pkg/log/core.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package log
```

Comment:
Can we stick with https://github.com/crossplane/crossplane-runtime/tree/master/pkg/logging instead?

Reply:
Done. I have removed the `log` package that I copied from wesaas repo.